cli_tools/gce_ovf_import/main.go

```diff
@@ -83,30 +83,14 @@ func buildImportParams() *ovfimportparams.OVFImportParams {
 		ShieldedIntegrityMonitoring: *shieldedIntegrityMonitoring, ShieldedSecureBoot: *shieldedSecureBoot,
 		ShieldedVtpm: *shieldedVtpm, Tags: *tags, Zone: *zoneFlag, BootDiskKmskey: *bootDiskKmskey,
 		BootDiskKmsKeyring: *bootDiskKmsKeyring, BootDiskKmsLocation: *bootDiskKmsLocation,
-		BootDiskKmsProject: *bootDiskKmsProject, Timeout: *timeout, Project: *project,
+		BootDiskKmsProject: *bootDiskKmsProject, Timeout: *timeout, Project: project,
 		ScratchBucketGcsPath: *scratchBucketGcsPath, Oauth: *oauth, Ce: *ce,
```

Comment:
Can you explain the history of the bug a bit, and why this fixes it?

Reply:
"project" is a input param. However, in many cases, user didn't type in this value (so it will be initialized as an empty value). Thus, the 'project' will be populated after the tool is lunched. We need that populated value. A pointer gives us the ability to reflect the updated 'project' in log.

Reply:
I'm not a big fan of updating a field via a pointer as it's hard to follow for others since it's implicit. Having an explicit update would be easier to follow and understand for anyone working on this code base in the future.

Reply:
I totally understand. This patch is to resolve our current problem of excluding test project in order to unblock our failure analysis - it's a fast fix. I have verified it works well and won't break.
I will work on a redesign to avoid implicit update soon.